alpha/apps/kaltura/lib/batch2/bcdl/kBusinessPreConvertDL.php

```diff
@@ -541,6 +541,8 @@ public static function decideAddEntryFlavor(BatchJob $parentJob = null, $entryId
 			$flavorAsset->save();
 		}
 
+		self::setAudioDescriptionLabel($flavorAsset, $flavorParams);
```

Comment:
The type of the second argument that should be sent to setAudioDescriptionLabel is a type that supports the function getMultiStream and getTagsArray.
In this function the variable $flavor needs to be sent